stratum/hal/lib/phal/tai/tai_optics_datasource.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020-present Open Networking Foundation
+// Copyright 2020 PLVision
+// SPDX-License-Identifier: Apache-2.0
+
+#include "stratum/hal/lib/phal/tai/tai_optics_datasource.h"
+
+#include "stratum/glue/integral_types.h"
+#include "stratum/glue/status/status.h"
+#include "stratum/glue/status/statusor.h"
+#include "stratum/hal/lib/common/common.pb.h"
+#include "stratum/hal/lib/phal/datasource.h"
+#include "stratum/hal/lib/phal/phal.pb.h"
+#include "stratum/lib/macros.h"
+
+namespace stratum {
+namespace hal {
+namespace phal {
+namespace tai {
+
+/*
+ * Creates a new TAI optics data source.
+ */
```

Comment:
```suggestion
// Creates a new TAI optics data source.
```

Reply:
Also, use either `/**/` or `//`. [Not both](https://google.github.io/styleguide/cppguide.html#Comment_Style)